### DISCUSSION TITLE
Proposal to adjust the operation of the S meter band

### DISCUSSION THREAD

**Discussion by Yurij20** [PE5PVB/TEF6686_ESP32#497] [Ideas]
There is a proposal to correct the operation of the S meter indicator strip.  Now, for example, in the absence of a signal at a value of -14 dbV, the S meter band does not decrease completely, but freezes at position “1”, which I consider not a very reliable indication.  For example, in firmware 1.18 onmytime, the S dead band completely decreases at 0 dbV.  I think it is necessary to correct the absolute “0” of the S meter band at a reception level value of -10 dbV, or even better, add a parameter for calibrating the S meter band to the menu.

**Comment by PE5PVB** [comment]
You are right. I did some modifications on it. Please check.

**Comment by Yurij20** [comment]
I checked it carefully, but it seems that the threshold for the indicator strip to appear is very late at 5 dbv.  Need at least 0 dbv

**Comment by PE5PVB** [comment]
OK. I can go down to 0dBuV. But the 0 doesn't mean that it's nothing, 0dBuV is just that the signal is 0dB different from 1uV, so signalstrength is 1uV. I set the initial value a bit higher because most have noise above 0dBuV.

**Comment by Yurij20** [comment]
better at 0dBV indeed.  It just turns out that the radio station at 6dBV is already clearly audible in stereo, but there is no band yet.  somehow not natural

**Comment by Yurij20** [comment]
I also think it’s necessary to move the red part of the bar to level +10.  Otherwise, starting at position 9 signals too early that the level is too high.  Even on the markings under lane 9 it is marked in green.  I also consider it necessary to shift the red part of the modulation meter band from the current 70 to 80
